### PR TITLE
[3562][MIG] hr_security_adj: Migration to 15.0

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
[3562](https://www.quartile.co/web#id=3562&model=project.task&view_type=form&menu_id=517)

### CODE REVIEW

**Review by kanda999 — CHANGES_REQUESTED** [review]
You have given the operator write access to hr.employee in this module but you can't update it. could you please check?
2023-07-10_15h11_15

File: hr_security_adj/models/hr_employee.py
Comment (kanda999) [inline hr_security_adj/models/hr_employee.py:12]: Why did you add this part? Im not sure that background.

Comment (AungKoKoLin1997) [reply hr_security_adj/models/hr_employee.py:12]: In odoo15, there are two views, employee public and employee. For employee public, create and write options are blocked by view definition and it's model is hr.employee.public and use for public view. So, we need to use normal employee view and this views is restricted only for hr user(https://github.com/odoo/odoo/blob/8221bef0a023d33476bdaca868f2ff0991f12a0d/addons/hr/views/hr_views.xml#L18-L30) and barcode field is used as domain in this view. And barcode is allowed only for hr user. So, when we open with hr operator, we will get error. That's why we need to new operator group to barcode field.

Comment (kanda999) [reply hr_security_adj/models/hr_employee.py:12]: I see. Please add a comment so we don't get confused when we find it later.

**Comment by kanda999** [comment]
And I get this error.  Could you update PR pleasae and make sure that the Operator is able to update the hr.employee record.
2023-07-11_11h06_17

File: hr_security_adj/views/hr_views.xml
Comment (kanda999) [inline hr_security_adj/views/hr_views.xml:6]: Why is this part necessary?

Comment (AungKoKoLin1997) [reply hr_security_adj/views/hr_views.xml:6]: employee view is only allowed to show for hr_group_user. So, we need to add operator group.https://github.com/odoo/odoo/blob/8221bef0a023d33476bdaca868f2ff0991f12a0d/addons/hr/views/hr_views.xml#L18-L30

**Review by yostashiro — APPROVED** [review]

**Review by kanda999 — APPROVED** [review]
Function review

**Pull request merged**
